competitions/LLIIA2020/ansible/roles/LLIIA2020/tasks/main.yml

```diff
@@ -17,3 +17,16 @@
     marker: "## {mark} ANSIBLE PLUGABLE AUTH OVERRIDE CONFIG"
     block: |
       $wgPluggableAuth_EnableLocalLogin = true;
+
+- name: Install additional tab configurations
+  blockinfile:
+    path: "{{ mediawiki_install_directory }}/mediawiki-1.33.0/LocalSettings.php"
+    marker: "## {mark} ANSIBLE ADDITIONAL TAB CONFIG"
+    block: |
+      # Overwrite the whole variable since LLIIA2020 is unique in handling of tabs
```

Comment:
Not that this is a good excuse: but this is copypasta from another project that has unique handling of tabs.

Ultimately it is unique in handling tabs in the sense that we would like to override the tabs, so this is auto-documented.

(I believe what is special is that we want these tabs instead of the default tabs)

Reply:
Maybe the important thing to clarify here is that "tab" in this case refers to MediaWiki page tabs, not TAB characters in a source file?  (Uh, I've got that right, right?  English only has a maximum of 17576 three-letter words, far fewer if one discards impossible combinations of consonants; I'm not surprised that we ran across one of the remaining words being overloaded!)